Procfile

```diff
@@ -1 +1,2 @@
 web: bundle exec unicorn -p $PORT -c ./config/unicorn.rb
+worker: bundle exec sidekiq -c 3 -t 25 -r ./whedon.rb
```

Comment:
As I mentioned, we're not actually using the worker right now but in theory we'd want this process.